app/src/main/res/values-nl/strings.xml

```diff
@@ -97,6 +97,13 @@
     <string name="contact_set_name">Geef contact een naam</string>
     <string name="contact_exists_message">Contact bestaat al, geef aub een unieke naam</string>
 
+    <!-- NFC -->
+    <string name="descr_gif">Dit zou een gif moeten zijn die nfc connection uitbeeldt.</string>
```

Comment:
Okay, it kinda does (I don't like the style tho, but thats another topic 🙂)

Could you still change the name tho 😇